### PR TITLE
Allow setting team for alerting in hieradata profile

### PR DESCRIPTION
**Pull request by bastjan**
## Checklist

- [x] Keep pull requests small so they can be easily reviewed.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog

### CODE REVIEW

File: modules/vshn-lbaas-hieradata/variables.tf [hunk 1]
```diff
@@ -71,3 +71,9 @@ variable "router_vip" {
   type        = string
   description = "Floating IP for the cluster's ingress controller/application router"
 }
+
+variable "team" {
+  type        = string
+  description = "Team to assign the load balancers to in Icinga. All lower case."
+  default     = ""
```
Comment (simu) [inline modules/vshn-lbaas-hieradata/variables.tf:78]: The variable can and should be mandatory for `vshn-lbaas-hieradata`:

```suggestion
```

File: modules/vshn-lbaas-cloudscale/variables.tf [hunk 2]
```diff
@@ -83,3 +83,9 @@ variable "internal_vip" {
   description = "Virtual IP for the Kubernetes/OpenShift API in the internal network"
   default     = ""
 }
+
+variable "team" {
+  type        = string
+  description = "Team to assign the load balancers to in Icinga. All lower case."
+  default     = ""
```
Comment (simu) [inline modules/vshn-lbaas-cloudscale/variables.tf:90]: I'd prefer making this mandatory as well, we can always consider making it optional in the Terraform which uses this variable.

```suggestion
```

File: modules/vshn-lbaas-hieradata/templates/hieradata.yaml.tmpl [hunk 3]
```diff
@@ -34,3 +34,7 @@ profile_openshift4_gateway::backends:
 %{ for be in backends["router"] ~}
     - ${be}
 %{ endfor ~}
+%{ if team != "" ~}
```
Comment (simu) [inline modules/vshn-lbaas-hieradata/templates/hieradata.yaml.tmpl:37]: If we make the variables mandatory, I'd still keep this check, as I don't think we want to require the variables to be non-empty.

**Review by simu — CHANGES_REQUESTED** [review]

File: modules/vshn-lbaas-cloudscale/variables.tf (hunk 2 above)
Comment (bastjan) [reply modules/vshn-lbaas-cloudscale/variables.tf:90]: Good point!

**Pull request merged**